### PR TITLE
Add Croatian, Bosnian, Serbian, Tigrinya pluralspec

### PR DESCRIPTION
**Pull request by dalu**
This also works for Bosnian and Serbian, but have not been added yet.

Not sure if tests are ok

### CODE REVIEW

**Comment by dalu** [comment]
bosnian, serbian copy/pasted and test tests all 3 language specs in 1 test function

**Comment by dalu** [comment]
hold on, I'll add Tigrinya (Eritrean, Aethiopian)

File: i18n/language/pluralspec_test.go [hunk 1]
```diff
@@ -512,6 +516,52 @@ func TestUkrainian(t *testing.T) {
 	runTests(t, "uk", tests)
 }
 
+func TestCroatianBosnianSerbian(t *testing.T) {
```
Comment (nicksnyder) [inline i18n/language/pluralspec_test.go]: I would prefer to have 3 top level test functions, one for each language, so the number of tests matches the number of langugages. You can write a helper method that just does the same checks for each of the 3 tests.

**Comment by dalu** [comment]
Your requested changes were added the same day btw.
Are you going to merge this or should I close the PR?

**Comment by nicksnyder** [comment]
Sorry for the delay. Apparently github does not notify me of commits, but I do get notified for comments on the PR.

**Comment by dalu** [comment]
Thanks :)

**Pull request merged**